haiku-data/wallpaper_community/wallpaper_community-0.1.recipe

```diff
@@ -2,7 +2,10 @@ SUMMARY="Wallpapers from the Haiku community"
 DESCRIPTION="Wallpapers created by members of the Haiku community"
 HOMEPAGE="https://github.org/haiku/artwork"
```

Comment:
I wonder if using https://www.haiku-os.org/news/2020-10-04-wallpaper-contest/ would be better?